### ISSUE TITLE
Missing information in pzmap function

### ISSUE THREAD

**Issue by lucasmsoares96** [gnu-octave/pkg-control#3]
First of all, you guys are doing an amazing job!
I'm having problems with the pzmap function. In matlab this function draws more information than in octave.

pzmap not draw the unit circle 

```octave
z = tf('z',1);
H = z / (((z-1)^2) * (z+0.5));
pzmap(H)
```

in matlab:
<img src="https://user-images.githubusercontent.com/60808407/234984317-fdea9fe4-f89e-4f40-be38-429c35009371.png"  width="500">

in octave:
<img src="https://user-images.githubusercontent.com/60808407/234984488-334fd0b9-8354-409e-a55a-9ea0ed8983d3.png"  width="500">

```octave
H = tf([2 5 1],[1 3 5]);
pzmap(H)
grid on
```

in matlab:
<img src="https://user-images.githubusercontent.com/60808407/234985507-52c55c69-c0c3-401d-a3e3-77f81e9ac0ee.png"  width="500">

in octave:
<img src="https://user-images.githubusercontent.com/60808407/234985668-23400666-fc72-4a07-812c-efebdd7bd1cc.png"  width="500">

**Comment by ttl-octave** [comment]
For the pole/zero map of a continuous-time system, you can use `sgrid` in order to get the desired grid for damping and eigenfrequency. The related function for discrete-time maps , `zgrid` is not yet implemented. Extending `grid` by automatically adding the damping/eigenfrequency grids for pole/zero maps is currently not planned.

Adding the axes or the unit circle for continuous- or discrete-time systems resp. should be straightforward and probably done for the next release.

**Comment by ttl-octave** [comment]
Axes and unit circle (discrete-time) are added by commit dcf3a5eb779265880cc4c0594297d484917eac12

**Comment by ttl-octave** [comment]
`zgrid` was implemented in version 3.6.0. Closing this issue.
